packages/web/index.js

```diff
@@ -23,6 +23,7 @@ const NEUTRINO_MODULES = join(__dirname, '../../node_modules');
 
 module.exports = (neutrino, opts = {}) => {
   const publicPath = './';
+  const extract = opts.extract || process.env.NODE_ENV === 'production';
```

Comment:
I'm not sure I like `extract` being an option for the web middleware, as it is completely specific to the style-loader. I think I'd rather enable/disable this via `opts.style.extract`.

Also, why are we only extracting in production? That seems off to me.
Either way, the way we have style-loader configured, HMR works, and the files aren't actually output.

If anything, I would the test should be `neutrino.options.command === 'build'`, but I kinda think we shouldn't do it conditionally. What do you think?

Reply:
> I'm not sure I like `extract` being an option for the web middleware, as it is completely specific to the style-loader. I think I'd rather enable/disable this via `opts.style.extract`.

Makes sense to me.

> Also, why are we only extracting in production?

It's unnecessary overhead during development. We can produce a build faster if we don't have to do extraction, and save extraction for when it really matters: when creating a production build.

> If anything, I would the test should be `neutrino.options.command === 'build'`, but I kinda think we shouldn't do it conditionally. What do you think?

I think it should be based on same condition of minification and chunking, which occur when in production mode, not for the build command. Right now production mode is synonymous with optimized mode, which I think extraction should fall under.